### PR TITLE
ENCD-3364  fixed text and link button misalignment

### PR DESCRIPTION
**Pull request by zoldello**
This is a tricky one. I took out the flex changes made by a prior developer. I could not see a strong reason why it was  needed in the first place and no other code seemed to use it.

### CODE REVIEW

**Comment by forresttanaka** [comment]
Does this branch mix ENCD-3883 and ENCD-3364?

**Comment by zoldello** [comment]
@forresttanaka I am normally careful when changing branches locally and checking out branches. I added a commit to undo the ENCD-384 commit (biosample.js file involving PMI.) Now, only one file should be modified.

**Comment by zoldello** [comment]
@forresttanaka: I will correct it so that the buttons are flushed.

File: src/encoded/static/scss/encoded/modules/_visualize.scss [hunk 1]
```diff
@@ -46,20 +46,21 @@
...
         font-weight: bold;
         font-size: 1.1rem;
+        width: 100px;
+        word-wrap: break-word;
     }
 
     @at-root #{&}__assembly-title {
         @extend .browser-selector__assembly;
         font-size: 1.3rem;
+
```
Comment (caseylitton) [inline src/encoded/static/scss/encoded/modules/_visualize.scss]: This looks unnecessary too, right?  For now let's air on the side of "best practice" and make as little changes as possible.

**Comment by zoldello** [comment]
@caseylitton: Removed space added unnecessarily.

**Pull request merged**